Add CyberPanel extension

xmok opened the pull request:
## Description

This is a Raycast extension for [CyberPanel](https://cyberpanel.net/). With this extension you can manage Users, Packages, Websites, Child Domains, Backups, Databases, Email Accounts, DNS Records and FTP Accounts in CyberPanel using CyberPanel Cloud API ([documentation link](https://documenter.getpostman.com/view/2s8Yt1s9Pf?version=latest)).

## Screencast

For ease of review, following are individual recordings of the different commands in action:

https://github.com/raycast/extensions/assets/30526133/14f4a02c-7885-4f0a-b2bd-b430a9b088d4

https://github.com/raycast/extensions/assets/30526133/29a9da68-e51a-46ab-8d24-bab03a54b8a5

https://github.com/raycast/extensions/assets/30526133/ef74d79c-4571-4073-b2ba-57ae534cf4fb

https://github.com/raycast/extensions/assets/30526133/85984739-9283-467b-9ed5-51d0f0f237c9

https://github.com/raycast/extensions/assets/30526133/e6c39829-b1d5-4bb2-a785-96ac3b13ad51

https://github.com/raycast/extensions/assets/30526133/1306ee0f-c0df-4c05-b834-f13efc322471

https://github.com/raycast/extensions/assets/30526133/2d7047b3-4299-4391-9c95-46d674f0ca4f

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer commented:
I would maybe add a tooltip in the list-users when you're creating a new user to explain what "Selected ACL" is 🙂

When you do that, can you fix my linting errors too 🙂

xmok commented:
> I would maybe add a tooltip in the list-users when you're creating a new user to explain what "Selected ACL" is 🙂
> 
> When you do that, can you fix my linting errors too 🙂

1. A user can modify ACL using the CyberPanel Dashboard but I've added a tooltip and placeholder as a reference:

<img width="1000" alt="cyberpanel-tooltip-for-acl" src="https://github.com/raycast/extensions/assets/30526133/c28ef251-d1c2-46bf-9a2d-8337251842c3">

2. In Packages, if bandwidth, disk space or domains equals ZERO, it means "Unlimited" rather than "zero" so i've added placeholder and better handling for that as well:
<img width="1000" alt="cyberpanel-placeholder-for-zero-equals-unlimited" src="https://github.com/raycast/extensions/assets/30526133/c1934664-1905-4319-a6e3-0baa441ab9bd">
<img width="1000" alt="cyberpanel-write-unlimited-instead-of-zero-for-some-values" src="https://github.com/raycast/extensions/assets/30526133/a0bb31e2-0035-401c-b10e-db87688b1b68">

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
